src/components/common/Select.js

```diff
@@ -16,9 +16,9 @@ const Select = ({ title, options, ...props }) => (
     <h4>{title}</h4>
 
     <SelectBox size="large" {...props}>
-      {options.map(option => (
-        <Option key={option} value={option}>
-          {option}
+      {options.map(({ version, release }) => (
+        <Option key={version} value={version}>
+          {release ? `${release} (${version})` : version}
```

Comment:
Perhaps we could make this the equivalent of ``release && version ? `${release} (${version})` : (release ? release : version)``

Reply:
never main, been explained :grin: